doc/env/bin/mkenv.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+
+from pathlib import Path
+import subprocess
+import sys
+import venv
+
+base = Path(sys.argv[1])
+env = Path(sys.argv[2])
+
+venv.create(env, with_pip=True)
+subprocess.run(
+    [
+        env / 'bin/pip',
```

Comment:
Would this work on Windows? Shall we use the path separator operator `/` rather than `"/"`?

```suggestion
        env / 'bin' / 'pip',
```

Reply:
Looks like it should do the right thing:

```console
>>> str(PureWindowsPath('c:\\abc') / 'def/ghi')
'c:\\abc\\def\\ghi'
```